__tests__/OmnichannelChatSDK.node.spec.ts

```diff
@@ -22,6 +22,37 @@ describe('Omnichannel Chat SDK (Node)', () => {
         jest.clearAllMocks();
     });
 
+    it('ChatSDK.startChat() with sendDefaultInitContext should not work on non-browser platform', async () => {
+        const chatSDK = new OmnichannelChatSDK(omnichannelConfig);
+        chatSDK.getChatConfig = jest.fn();
+
+        await chatSDK.initialize();
+
+        chatSDK.IC3Client = {
+            initialize: jest.fn(),
+            joinConversation: jest.fn()
+        }
+
+        const optionalParams = {
+            sendDefaultInitContext: true
+        }
+
+        jest.spyOn(chatSDK.OCClient, 'getChatToken').mockResolvedValue(Promise.resolve({
+            ChatId: '',
+            Token: '',
+            RegionGtms: '{}'
+        }));
+
+        jest.spyOn(chatSDK.OCClient, 'sessionInit').mockResolvedValue(Promise.resolve());
+
+        const errorMessage = 'UnsupportedPlatform';
+        try {
+            await chatSDK.startChat(optionalParams);
```

Comment:
can we add an error message after start chat,  in case the call dont throw an error?, for all the similar scenarios